### ISSUE TITLE
Test bench

### ISSUE THREAD

**Issue by joshlvmh** [geoscience-community-codes/tephra2#4]
Hi,

I was wondering if there was a test bench or similar available that was used during the creation of tephra2 that verifies expected outputs. I am trying to vectorise some of the loops within the source code and wanted to make sure the output is still correct.

Thank you.

**Comment by ljc-geo** [comment]
I don't think so, There has been lots of editing and testing during code creation. Exactly what kind of data are you looking for?

**Comment by joshlvmh** [comment]
Something that tests the expected outputs of key functions, or instead a set of examples testing edge cases and extremes and then an acceptable threshold for comparing subsequent runs to the expected output? Thank you for your reply!
